projects/optic/src/commands/capture/capture.ts

```diff
@@ -131,6 +139,31 @@ const getCaptureAction =
       return;
     }
 
+    if (options.init) {
+      // no optic.yml is present but the command would attempt to write
+      if (!config.configPath && !options.stdout) {
```

Comment:
If this is the case we should write to `config.root` - this is either the git root, or the cwd